### PR TITLE
[SEO] Add Ayatal Kursi slugs

### PR DESCRIPTION
**Pull request by osamasayed**
### Summary
This PR makes it possible to access Ayatul Kursi via the slugs `/ayatul-kursi` or `/آیت الکرسی`.

### CODE REVIEW

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**/\[chapterId\]**](https://quran-com-i6akwt19n-qurancom.vercel.app/[chapterId]) |
|:-:|
|<a href="https://quran-com-i6akwt19n-qurancom.vercel.app/[chapterId]">Screenshot of /[chapterId]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/DQegB47XOToVpzYfw4useOy_Z7kNRGVb5VLFnhK2phlcLHiMiiD2_lc5bd5Kf_oFCntoJrgJ-jof6uA-brytUw.png">(view full size)</a> |







Commit f088b0ab6e59bdfa8c241a66e960e5712dc3e079 (https://quran-com-i6akwt19n-qurancom.vercel.app).

File: src/pages/[chapterId]/index.tsx [hunk 1]
```diff
@@ -54,20 +58,27 @@ const Chapter: NextPage<ChapterProps> = ({
...
+    if (AYAH_KURSI_SLUGS.includes(chapterIdOrVerseKeyOrSlug.toLowerCase())) {
+      chapterIdOrVerseKeyOrSlug = '2:255';
+    } else {
+      const sluggedChapterId = await getChapterIdBySlug(chapterIdOrVerseKeyOrSlug, locale);
+      // if it's not a valid slug
+      if (!sluggedChapterId) {
+        return { notFound: true };
+      }
+      chapterId = sluggedChapterId;
+      isChapter = true;
```
Comment (muhajirdev) [inline src/pages/[chapterId]/index.tsx:80]: nit: the `if`s a bit too nested.

Is it possible to reduce the number of nested `if's ? 

for example
```
if (!isChapter && !isValidVerseKey(chapterIdOrVerseKeyOrSlug)) {
     // if the value is a slug of Ayatul Kursi
     if (AYAH_KURSI_SLUGS.includes(chapterIdOrVerseKeyOrSlug.toLowerCase())) {
       chapterIdOrVerseKeyOrSlug = '2:255';
```

maybe could  be

```
check if the value is not chapter, not a verse key, and not ayatul curse
if (!isChapter && !isValidVerseKey(chapterIdOrVerseKeyOrSlug) && AYAH_KURSI_SLUGS.includes(chapterIdOrVerseKeyOrSlug.toLowerCase())) {
       chapterIdOrVerseKeyOrSlug = '2:255';

```

or the better way, we can turn it into a function so it's easier to understand.

File: src/pages/[chapterId]/index.tsx [hunk 2]
```diff
@@ -40,10 +40,14 @@ const Chapter: NextPage<ChapterProps> = ({
...
-          lang,
-        )}-${toLocalizedNumber(chapterResponse.chapter.versesCount, lang)}`}
+        title={`${t('surah')} ${chapterResponse.chapter.transliteratedName} - ${
+          isChapter
+            ? `${toLocalizedNumber(1, lang)}-${toLocalizedNumber(
+                chapterResponse.chapter.versesCount,
+                lang,
+              )}`
+            : `${versesResponse.verses[0].verseNumber}`
+        }`}
```
[the remark this answers is not in the document]
Comment (osamasayed) [reply src/pages/[chapterId]/index.tsx:50]: You have a point. As per my discussion with Abdellatif, I will have a PR to clean up this file inshAllah.

**Comment by mmahalwy** [comment]
#### 📝 Changed routes:


| [**/\[chapterId\]**](https://quran-com-1asgk11q0-qurancom.vercel.app/[chapterId]) |
|:-:|
|<a href="https://quran-com-1asgk11q0-qurancom.vercel.app/[chapterId]">Screenshot of /[chapterId]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/_IYpcgf5UIw22TlaJfCyeLduSrPDSlHoGh109PgIJb3daEPP8LCOwbjhXzAh7-t8BEUsFO64a2J95sjlhtNK8A.png">(view full size)</a> |







Commit e95a1ae8de387a8a9e100f7902320ce396fc276a (https://quran-com-1asgk11q0-qurancom.vercel.app).

**Pull request merged**